CONTRIBUTING.md

```diff
@@ -105,40 +115,45 @@ Avoid commenting code. A lot of the code that is commented, never gets uncomment
 With the function that we improved above there is still a glaring issue :
 
 ```
-void Commands::setTto(bool throttleTimeOut) {
-    String msg = "ASTRO Throttle Timeout " + String(throttleTimeOut ? "On" : "Off");
-    println(msg);
-
+void Commands::acc(bool hasAccelerationLimit) {
+    for (i = 0; i < RobotMotor::numMotors; i++) {
+        motorList[i].accLimit = hasAccLimit;
+        String msg = "ASTRO Motor " + String(i + 1);
+        msg += " Acceleration Limiter: ";
+        msg += String(motorList[i].accLimit ? "Open" : "CLose");
+        println(msg);
+    }
 }
 ```
 
-What the heck is `TTO`? By observing this function and it's argument, it is obvious that it's for `Throttle Timeout`. However, imagine that you were stumbling upon a call to this function like this
+What the heck is `acc`? By observing this function and it's argument, it is obvious that it's for `Acceleration Limit`. However, imagine that you were stumbling upon a call to this function like this
 
 ```
 //some code
-setTto(false);
+acc(false);
 //more code
 ```
-To know what `TTO` means, you need to go to the function definition which could be tedious if you have to do this often.
-
-Furthermore, you would expect `throttleTimeOut` to be a number, the timeout time, but it is a boolean.
+To know what `acc` means, you need to go to the function definition which could be tedious if you have to do this often.
 
 ```
-void Commands::hasThrottleTimeout(bool hasThrottleTimeout) {
-    String msg = "ASTRO Throttle Timeout " + String(hasThrottleTimeout ? "On" : "Off");
-    println(msg);
+void Commands::setAccelerationLimit(bool hasAccelerationLimit) {
+    for (i = 0; i < RobotMotor::numMotors; i++) {
+        motorList[i].accLimit = hasAccLimit;
+        String msg = "ASTRO Motor " + String(i + 1);
+        msg += " Acceleration Limiter: ";
+        msg += String(motorList[i].accLimit ? "Open" : "CLose");
+        println(msg);
+    }
 }
 ```
 
-Boolean variable should start with `has` or `is`.
-
+Now it is much cleaner when the call is being read.
 
 ```
 //some code
-hasThrottleTimeout(false);
+setAccelerationLimit(false)
```

Comment:
Missing semi-colon, though it is just for example sake we might as well keep it syntactically correct.

Reply:
Yup will fix, got used to JS and Python too much